File: lib/Tiquete.dart
class Tiquete {
  int idPago;
  int cedulaCliente;
  String descripcipon;
  String fecha;
  String numeroSerieBus;
  int costo;
  String LugarLlegada;
  String LugarSalida;

  Tiquete(
      {required this.idPago,
      required this.cedulaCliente,
      required this.descripcipon,
      required this.fecha,
      required this.numeroSerieBus,
      required this.costo,
      required this.LugarLlegada,
      required this.LugarSalida});

  int getIdPago() {
    return idPago;
  }

  int getCedulaCliente() {
    return cedulaCliente;
  }

  String getDescripcion() {
    return descripcipon;
  }

  String getFecha() {
    return fecha;
  }

  String getSerieBus() {
    return numeroSerieBus;
  }

  int getCosto() {
    return costo;
  }

  String getLugarLL() {
    return LugarLlegada;
  }

  String getLugarS() {
    return LugarSalida;
  }

  factory Tiquete.fromJson(Map<String, dynamic> json) => Tiquete(
      idPago: json['idPago'],
      cedulaCliente: json['cedulaCliente'],
      descripcipon: json['descripcipon'],
      fecha: json['fecha'],
      numeroSerieBus: json['numeroSerieBus'],
      costo: json['costo'],
      LugarLlegada: json['lugarLlegada'],
      LugarSalida: json['lugarSalida']);
}

File: API_FLUTTER/ApiFlutter/ApiFlutter/Models/Tiquete.cs
using System.ComponentModel.DataAnnotations;

namespace ApiFlutter.Models
{
    public class Tiquete
    {
        [Key]
        public int idPago { get; set; }
        [Required]
        public int CedulaCliente { get; set; }
        [Required]
        public string descripcipon { get; set; }
        [Required]
        public DateTime Fecha { get; set; }
        [Required]
        public string NumeroSerieBus { get; set; }
        [Required]
        public double Costo { get; set; }
        [Required]
        public string LugarLlegada { get; set; }
        [Required]
        public string LugarSalida { get; set; }
    }
}
